repo2docker/app.py

```diff
@@ -243,14 +247,20 @@ def initialize(self):
             # remove the additional newline from the stream handler
             self.log.handlers[0].terminator = ''
             # We don't want a [Repo2Docker] on all messages
-            self.log.handlers[0].formatter = logging.Formatter(fmt='%(message)s')
+            self.log.handlers[0].formatter = logging.Formatter(
+                fmt='%(message)s'
+                )
```

Comment:
shouldn't this closing paren be flush left?

Reply:
/shrug

I just put it where my editor put it and the linter didn't complain.